centaur/src/main/resources/standardTestCases/cwl_cache_between_workflows/cwl_cache_between_workflows.cwl

```diff
@@ -36,7 +36,7 @@ $graph:
 
   outputs:
     trapezoidalArea:
-      type: float
+      type: long
```

Comment:
Are we still testing `float` somewhere else ?

Reply:
Should be - `baseAverage` still uses `float` and and its value appears in the final outputs (would love a cross-check on this thinking from a CWL expert)

```
{
  "cwl-cache-between-workflows.trapezoidalArea": "77",
  "cwl-cache-between-workflows.baseAverage": 11.0
}
```

Reply:
Checking this now on branch [`aen_4023_testing_the_test`](https://github.com/broadinstitute/cromwell/compare/aen_4023_testing_the_test?expand=1) where I deliberately break `float`

Reply:
Yep, we get `java.lang.RuntimeException: unrecognized simpleton WOM type: Float` as expected